common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -201,7 +201,7 @@ trait CommercialSwitches {
     description = "Turn on Sonobi header bidding",
     owners = Seq(Owner.withGithub("rich-nguyen"), Owner.withGithub("janua")),
     safeState = Off,
-    sellByDate = new LocalDate(2017, 3, 31), //Friday
+    sellByDate = new LocalDate(2017, 4, 28), //Friday
```

Comment:
This one is not a test, we're not going to switch it off but I'll leave it to @rich-nguyen whether or not to enable it ad infinitum. The second one is a test but is central to work in Q1, so these changes are conservative.